### DISCUSSION TITLE
increase specifity for tailwind classes - use case: widget

### DISCUSSION THREAD

**Discussion by bairrada97** [tailwindlabs/tailwindcss#12474] [Help]
Im developing a widget and im trying to make it bulletproof against the parent website where the widget is gonna live.
I created  a  'fake css file' to simulate a parent website where they add !important to most of possible styles.
is there a way to override that using tailwind?

the following snippet works in case the parent website does not have any !important in their codebase. but im looking in case they add !important to their styles.

 ```
 module.exports = {
    content: ["./public/index.html", "./src/**/*.{js,ts,jsx,tsx}"],
    prefix: "app-",
    important: true,
  }
   ```
  
  example of what I want:
  
 ```
parent.css
   h4 {
     font-size: 40px !important
   }
   
my code should apply this class but is not
   <h4 className="app-text-20"></h4>
  
```

**Comment by wongjn** [comment]
`app-text-20` would not resolve to a valid class in Tailwind with the configuration you have provided. It should otherwise work as you describe for valid Tailwind classes, where the parent page's rule specificity rules are `0 1 0` or lower.
